gnucash/report/standard-reports/test/CMakeLists.txt

```diff
@@ -6,6 +6,9 @@ SET(scm_test_standard_reports_SOURCES
   test-standard-net-linechart.scm
 )
 
+set(scm_test_with_srfi64_SOURCES
```

Comment:
I believe the scm_test_with_srfi64_SOURCES variable should be added to the dist list at the end of this file (in SET_DIST_LIST(test_standard_reports_DIST ...))
The set is currently empty but it won't remain empty...

Reply:
Travis wouldn't detect this issue. It would only surface in a make/ninja-build dist-check, and even then only after you start adding unit test sources to the variable.
But since this is the commit in which you introduce the variable, I believe it would be most elegant to make it a complete introduction :)